packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -118,6 +118,7 @@ function runExec(opts, execCode) {
     transform: babel.transform,
     opts,
     exports: {},
+    require
```

Comment:
Ok cool, and we already have `let exec   = task.exec` which has the filename at `exec.filename`